R-package/R/callbacks.R

```diff
@@ -766,11 +766,12 @@ xgb.gblinear.history <- function(model, class_index = NULL) {
   enames <- names(eval_res)
   if (is.null(enames))
     stop('evaluation results must have names')
-  iter <- sprintf('[%d]\t', iter)
+  iter <- sprintf('[round: %d]\t', iter)
```

Comment:
I don't think this is necessary.

Reply:
The thing is that if you print them as `[<number>] <space>`  then they look quite similar to R's own printing of character vectors, which can be a bit confusing.

Reply:
That's probably fine for now, we would like to keep the output consistent across bindings at the moment. Besides, the output format is well-known for all other libraries.

Reply:
Removed the `round: ` part.

I think the python interface could also show a plusminus sign instead of plus. Since python2 is not supported, I don't think it should create any issue there.